chimney/.jvm/src/test/scala/io/scalaland/chimney/JavaEnumSpec.scala

```diff
@@ -0,0 +1,123 @@
+package io.scalaland.chimney
+
+import utest._
+import io.scalaland.chimney.examples.JavaEnums
+import io.scalaland.chimney.examples.JavaEnums.Colors3
+import io.scalaland.chimney.examples._
+
+object JavaEnumSpec extends TestSuite {
+
+  val tests = Tests {
+
+    "enum to enum" - {
+      "narrow to wide" - {
+        val t = Transformer
+          .define[JavaEnums.Colors3, JavaEnums.Colors6]
+          .buildTransformer
+
+        t.transform(JavaEnums.Colors3.Black) ==> JavaEnums.Colors6.Black
+        t.transform(JavaEnums.Colors3.Blue) ==> JavaEnums.Colors6.Blue
+        t.transform(JavaEnums.Colors3.Green) ==> JavaEnums.Colors6.Green
+        t.transform(JavaEnums.Colors3.Red) ==> JavaEnums.Colors6.Red
+      }
+
+      "wide to narrow auto-derivation fails" - {
+        compileError(
```

Comment:
In order to avoid false negatives, suggest to assert on fragment of compilation error using `compileError(...).check(...)`